apps/backend/src/routes/moments.ts

```
@@ -0,0 +1,199 @@
+import { Hono } from 'hono';
+import { drizzle } from 'drizzle-orm/d1';
+import { moments } from '@personal-hub/shared';
+import {
+  createMomentSchema,
+  updateMomentSchema,
+  momentQuerySchema,
+  type MomentResponse,
+} from '@personal-hub/shared';
+import { zValidator } from '@hono/zod-validator';
+import { verifyAuth } from '../lib/auth';
+import { eq, desc, and, like, or, sql } from 'drizzle-orm';
```

ESLint

[error] 'or' is defined but never used.

(@typescript-eslint/no-unused-vars)